### PR TITLE
docs: clarify browser support and add issue severity classification

### PR DESCRIPTION
**Pull request by rubencarvalho**
## Description

This PR adds a new section to the `CONTRIBUTING.md` and `README` files to provide contributors with clear guidelines on the target browsers we support. It also adds a detailed defect severity classification table into the contribution guidelines and adds a  "Issue Severity" dropdown field to the GitHub issues template.

## Related issue(s)

- resolves https://github.com/adobe/spectrum-web-components/issues/3465 
- SWC-52
- SWC-181

## Motivation and context

We are aiming for greater transparency while at the same time formalizing some of our current practices and support mechanisms.

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [x] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [x] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [x] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,6 +54,21 @@ Create an issue on the repository and provide the following information by filli
 -   What you would expect to happen
 -   What actually happens
 -   Notes (for example why you think this might be happening, or stuff you tried that didn't work)
+-   Issue severity
+
+### Issue severity classification
+
+Providing the correct issue severity classification allows us to more adequately assess and prioritize your issue.
```
Comment (najikahalsema) [inline CONTRIBUTING.md:61]:
```suggestion
Providing the correct issue severity classification helps us adequately assess and prioritize your issue.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -54,6 +54,21 @@ Create an issue on the repository and provide the following information by filli
...
+| **SEV 1**      | A critical design or functionality issue that breaks the design system, causes significant usability problems, or exposes critical security vulnerabilities. This issue impacts all users and/or essential workflows, making users unwilling or unable to complete any tasks. Requires immediate attention.                                           | Broken navigation system, complete unresponsiveness on all devices, components not rendering, inaccessible primary actions, security vulnerabilities. Accessibility: The end user is not able to access the product or complete core user tasks or activities (key navigational elements not accessible via keyboard, missing or incorrect form labels that prevent screen reader users from completing forms or actions, critical color contrast issues that prevent users from reading or interacting with essential content). |
```
Comment (najikahalsema) [inline CONTRIBUTING.md:67]:
```suggestion
| **SEV 1**      | A critical design or functionality issue that breaks the design system, causes significant usability problems, or exposes critical security vulnerabilities. This issue impacts all users and/or essential workflows, making users unable to complete any tasks. Requires immediate attention.                                           | Broken navigation system, complete unresponsiveness on all devices, components not rendering, inaccessible primary actions, security vulnerabilities. Accessibility: The end user is not able to complete core tasks or activities (key navigational elements not accessible via keyboard, missing or incorrect form labels that prevent screen reader users from completing forms or actions, critical color contrast issues that prevent users from reading or interacting with essential content). |
```
Minor wording adjustments.
Also, can we add a newline to the accessibility examples to make it easier to read?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -54,6 +54,21 @@ Create an issue on the repository and provide the following information by filli
...
+| **SEV 2**      | A significant problem affecting the design, usability or functionality of the system or components for a subset of users. These issues cause major disruptions and prevent users from completing tasks or workflows. Accessibility: Does not conform with WCAG 2.1 Level AA criteria and blocks core user tasks (no known workaround).                | Content that is out of view or unreachable by customers, critical buttons or links not functioning or actions that cannot be submitted. Unintentional breaking changes causing disruptions in production environments. Accessibility: Issue with a component or controller with widespread use and blocks core user tasks (no known workaround).                                                                                                                                                                                 |
```
Comment (najikahalsema) [inline CONTRIBUTING.md:68]:
```suggestion
| **SEV 2**      | A significant problem affecting the design, usability or functionality of the system or components for a subset of users. These issues cause major disruptions and prevent users from completing tasks or workflows. Accessibility: Does not conform with WCAG 2.1 Level AA criteria and blocks core user tasks (no known workaround).                | Content that is out of view or unreachable by customers, critical buttons or links not functioning, or actions that cannot be submitted. Unintentional breaking changes causing disruptions in production environments. Accessibility: Issue with a component or controller with widespread use and blocks core user tasks (no known workaround).                                                                                                                                                                                 |
```

Please add newline between different categories of examples, too.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -54,6 +54,21 @@ Create an issue on the repository and provide the following information by filli
...
+| **SEV 3**      | A design or functionality issue that causes noticeable errors or minor usability problems for users. These defects cause confusion or degrade user experience but do not prevent task completion. Accessibility: Does not conform with WCAG 2.1 Level AA criteria and is either non-blocking for core user tasks or blocking for non-core user tasks. | Misleading labels, inconsistent component behavior, unexpected interactions, decreases in system performance. Accessibility: Workarounds are available and discoverable for the end user to complete core user tasks, or the end user is not able to complete non-core user tasks (inadequate ARIA labels, improper focus management, insufficient color contrast for non-critical elements).                                                                                                                                    |
```
Comment (najikahalsema) [inline CONTRIBUTING.md:69]:
```suggestion
| **SEV 3**      | A design or functionality issue that causes noticeable errors or minor usability problems for users which either cause confusion or degrade the user experience, but do not prevent task completion. Accessibility: Does not conform with WCAG 2.1 Level AA criteria and is either non-blocking for core user tasks or blocking for non-core user tasks. | Misleading labels, inconsistent component behavior, unexpected interactions, decreases in system performance. Accessibility: Workarounds are available and discoverable for the end user to complete core user tasks, or the end user is not able to complete non-core user tasks (inadequate ARIA labels, improper focus management, insufficient color contrast for non-critical elements).                                                                                                                                    |
```
Minor tweak + newlines pleaseee

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -54,6 +54,21 @@ Create an issue on the repository and provide the following information by filli
...
+| **SEV 4**      | A minor design flaw that affects user experience but doesn't have a serious impact on overall usability or functionality. These issues do not prevent users from completing tasks. Accessibility: Does not conform with WCAG 2.1 Level AA criteria but has lower user impact.                                                                         | Minor visual inconsistencies, non-critical content misalignment, or minor layout issues. Accessibility: A WCAG violation is present in areas not frequently visited by end users or has a lower impact on overall accessibility and usability.                                                                                                                                                                                                                                                                                   |
```
Comment (najikahalsema) [inline CONTRIBUTING.md:70]:
```suggestion
| **SEV 4**      | A minor design flaw that affects user experience, but doesn't have a serious impact on overall usability or functionality. These issues do not prevent users from completing tasks. Accessibility: Does not conform with WCAG 2.1 Level AA criteria but has lower user impact.                                                                         | Minor visual inconsistencies, non-critical content misalignment, or minor layout issues. Accessibility: A WCAG violation is present in areas not frequently visited by end users, or it has a lower impact on overall accessibility and usability.                                                                                                                                                                                                                                                                                   |
```

Minor tweaks + request for newlines

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -92,6 +107,23 @@ type(scope?): subject #scope is optional, but should reference the package you a
 
 We maintain a specific standard of commit messages while committing to the respository. See [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
 
+## Browsers support
```
Comment (najikahalsema) [inline CONTRIBUTING.md:110]:
```suggestion
## Browser support
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -92,6 +107,23 @@ type(scope?): subject #scope is optional, but should reference the package you a
...
+
+When contributing to our system, please consider the target browsers we support.
```
Comment (najikahalsema) [inline CONTRIBUTING.md:112]:
```suggestion
When contributing to our system or filing an issue, please consider the target browsers we support.
```

**Review by najikahalsema — CHANGES_REQUESTED** [review]
Looking great so far. Since you're in here, could we also add some more specificity around using webcomponents.dev to provide your reproduction? 

in `bug_report.yaml`: 
line 65: "Sample code or abstract reproduction which illustrates the problem"

in `contributing.md` (it wouldn't let me comment on these lines for some reason)
lines 51-53
"Please reproduce your issue in an abstract environment, preferrably webcomponents.dev (use the existing link provided there). Screenshots or videos in your local dev environment are not sufficient enough to help us identify whether or not the issue originates from our library."

File: CONTRIBUTING.md (hunk 7 above)
Comment (rubencarvalho) [reply CONTRIBUTING.md]: Updated.

File: CONTRIBUTING.md (hunk 5 above)
Comment (rubencarvalho) [reply CONTRIBUTING.md]: Updated.

File: CONTRIBUTING.md (hunk 4 above)
Comment (rubencarvalho) [reply CONTRIBUTING.md]: Updated.

File: CONTRIBUTING.md (hunk 3 above)
Comment (rubencarvalho) [reply CONTRIBUTING.md]: Updated.

File: CONTRIBUTING.md (hunk 2 above)
Comment (rubencarvalho) [reply CONTRIBUTING.md]: Updated.

File: CONTRIBUTING.md (hunk 1 above)
Comment (rubencarvalho) [reply CONTRIBUTING.md]: Updated.

**Review by najikahalsema — CHANGES_REQUESTED** [review]
Just one little thing. Otherwise, looks good! Thank you.

**Review by najikahalsema — APPROVED** [review]
Woohoo! Thank you!!!

**Pull request merged**